website/pages/docs/troubleshooting.md

```diff
@@ -22,3 +22,21 @@ There are a couple of ways to get help for any CloudQuery-related issues or ques
 ### Verbose Logging
 
 Usually the first step that will be needed to debug/resolve an issue is to run `cloudquery` with `--log-level debug` to enable verbose logging.
+
+### Error: "failed to migrate source"...
+
+If you see an error such as `failed to migrate source`, it means that, while upgrading a plugin, the migration of the SQL schema failed.
+CloudQuery makes a best-effort attempt to automatically and transparently manage the schemas of plugins, but 
+this can sometimes fail during version upgrades.
+
+The easiest solution is to drop and recreate the database or schema (or all the plugins tables, such as `aws_*`).
+Some queries that may help you (handle with care!):
+
+```sql
+DROP SCHEMA public CASCADE;
+CREATE SCHEMA public;
```

Comment:
If you recreate the public schema don't you also need to regrant usage permissions?